### DISCUSSION TITLE
More customization

### DISCUSSION THREAD

**Discussion by subnwa** [ppy/osu#16863] [Ideas] [answered]
Desktop or other versions may have more customization options within the program. Of course, I don't know how these customizations happen, but people can try to achieve these customizations.

For example, it can be features such as background change, avatar frame.

**Comment by bdach** [comment, marked as answer]
This discussion is too vague to the point of bringing no value. Several "customization options" already exist in multiple facets of the game.

I would ask to familiarise yourself with what is already in place first before proceeding to suggest new functionality.

**Reply by subnwa** [reply]
sorry. I'll be more careful next time
